guides/doc-Upgrading_and_Updating/topics/upgrading_external_database.adoc

```diff
@@ -0,0 +1,25 @@
+[id="Upgrading_External_Database_{context}"]
+= Upgrading External Database
+
+You can upgrade an external database from {EL} 7 to {EL} 8 while upgrading {Project} from {ProjectVersionPrevious} to {ProjectVersion}.
+
+.Prerequisites
+. Create a backup.
++
+* On a virtual machine, take a snapshot.
+* On a physical machine, create a backup.
+. Create a new {EL} 8 based host for PostgreSQL server that follows the external database on {EL} 8 documentation.
```

Comment:
Can you add a link to that section so users don't need to search for it?

Reply:
@ekohl I am not 100% sure if this is the correct link but I am adding this:
(https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/deploying_different_types_of_servers/using-databases#migrating-to-a-rhel-8-version-of-postgresql_using-postgresql)